.metadocrc

```diff
@@ -1,6 +1,8 @@
 {
   "headerFile": "doc/header.md",
   "footerFile": "doc/footer.md",
+  "generateContentsTable": true,
+  "removeInterface": true,
```

Comment:
```suggestion
```
?

Reply:
Since there is only one interface, I'd prefer to not generate the header for it and IMO it looks better without interface header.

Reply:
I agree, but this at least shouldn't be a part of the same commit.